src/microsoft-trydotnet-editor/tests/monacoEditorSimulator.ts

```diff
@@ -13,9 +14,14 @@ export class MonacoEditorSimulator extends editorAdapter.EditorAdapter {
 
     private _options = {};
     private _code: string;
+    private _size?: { width: number; height: number; }
```

Comment:
Does this need to be optional?

Reply:
this is only internal state of test object, can be udnefined